integration-tests/src/test/java/org/alfresco/core/NetworksApiServiceIntegrationTest.java

```diff
@@ -0,0 +1,82 @@
+package org.alfresco.core;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import org.alfresco.core.handler.NetworksApiClient;
+import org.alfresco.core.handler.NodesApiClient;
+import org.alfresco.core.model.PersonNetworkEntry;
+import org.alfresco.core.model.PersonNetworkPaging;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.shaded.org.apache.commons.lang.RandomStringUtils;
+
+@Testcontainers
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class NetworksApiServiceIntegrationTest {
+
+    @Container
+    public static DockerComposeContainer compose = new DockerComposeContainer(new File("src/test/resources/docker-compose.yml"))
+        .withLocalCompose(true)
+        .withExposedService("alfresco", 8080, Wait.forListeningPort().withStartupTimeout(Duration.ofSeconds(90)));
+
+    @Autowired
+    private NetworksApiClient networksApiClient;
+
+    @Autowired
+    private NodesApiClient nodesApiClient;
+
+    public static String getRandomNodeName() {
+        return "test-node-" + RandomStringUtils.randomNumeric(8);
+    }
+
+    @BeforeAll
+    public void setUp() {
+        long awaitTimeoutInMilliseconds = 120000;
+        await()
+            .atMost(awaitTimeoutInMilliseconds, TimeUnit.MILLISECONDS)
+            .pollDelay(100000, TimeUnit.MILLISECONDS)
+            .untilAsserted(() ->
+                assertThat(nodesApiClient
+                    .getNode("-my-", null, null, null)
```

Comment:
-my- => Constant, maybe it's even worth to have a CommonConstants class in tests where you have it as public

Reply:
done